rust/ql/test/extractor-tests/generated/Function/CONSISTENCY/DataFlowConsistency.expected

```diff
@@ -1,2 +0,0 @@
-uniqueEnclosingCallable
```

Comment:
This (and other) files can be removed now. I have used this script locally for removing empty consistency files:
```
find rust/ql/test -name "*Consistency.expected" -size 0 -print -delete
```

Reply:
Nice, thanks 🙏